s31052-LukaszBurnicki/zajecia1/zadania1-14

```diff
@@ -0,0 +1,58 @@
+1. Uruchom terminal
+	Uruchomiony
+
+2. Wyświetl zawartość aktualnego katalogu
+	ls
+
+3. Sprawdź gdzie aktualnie się znajdujesz
+	pwd
+
+4. Przejdź do katalogu Pulpit za pomocą ściezki względnej
+	cd Desktop/
+
+5. Sprawdź gdzie jesteś
+	pwd
+
+6. Wyświetl zawartość bieżącego katalogu
+	ls
+
+7. Wyświetl zawartość swojego katalogu domowego bez przechodzenia do niego
+	ls ~
+
+8. Przejdź do katalogu Obrazy (lub innego)
+w twoim katalogu domowym za pomocą ścieżki względnej
+	cd Pictures
+
+9. Sprawdź gdzie jesteś
+	pwd
+
+10. Zobacz do jakiego katalogu przejdziesz za pomocą ścieżki ././././././
+W opisie rozwiązania proszę umieścić również to jak rozumiesz co
+się właściwie stało.
+Dlaczego znalazła/eś się w tym miejscu do którego przeniosła Cię
+ta ścieżka?
+Czy dodanie kilku ./././././ na początku/środku/końcu powoduje,
+że ścieżka się zmieni?
+Co ta ścieżka oznacza/sobą reprezentuje?
+	Przez kropkę na początku nigdzie się nie ruszę.
```

Comment:
- Już gdzieś widziałem taką odpowiedź
- Nie o to chodziło w tym zadaniu a o merytoryczne zrozumienie znaczenia znaku pojedynczej kropki w zadaniu a nie jej usuwanie z początku.
- Do poprawy